wildlifelicensing/apps/applications/models.py

```diff
@@ -60,6 +60,16 @@ class Application(RevisionedMixin):
         ('not_reviewed', 'Not Reviewed'), ('awaiting_amendments', 'Awaiting Amendments'), ('amended', 'Amended'),
         ('accepted', 'Accepted'))
 
+    TYPE_NEW_LICENCE = 'new licence'
+    TYPE_AMENDMENT = 'amendment'
+    TYPE_RENEWAL = 'renewal'
+    APPLICATION_TYPE_CHOICES = (
+        (TYPE_NEW_LICENCE, TYPE_NEW_LICENCE.title()),
+        (TYPE_AMENDMENT, TYPE_AMENDMENT.title()),
+        (TYPE_RENEWAL, TYPE_RENEWAL.title()),
+    )
```

Comment:
We didn't do this style (i.e. declare types) for processsing/customer statuses, so maybe keep it the same here.